free-2.md

```diff
@@ -52,18 +61,17 @@ This section examines whether the idea makes for a good CS 111 project.
 
 
 ### Suitability
-_If someone were to work on this project, what percentage of their time would be
-spent directly engaging in the **language** aspects of this project (e.g.,
-making language design decisions), as opposed to "systems" aspects of the
-project (e.g., implementing a complicated semantics that doesn't require a lot
-of language design)?_
+I think time would be evenly distibuted implementing the system and designing
+the language. 
 
 
 ### Scope
-_How big an idea is this? How ambitious is this project?_
+I do not think this project is very ambitious. I would only need a basic knowledge
+of how to set up an environment in the given operating system. The language itself
+could probably be written in Scala. 
```

Comment:
That could be harder than you think, since as far as I know the kernel tends to take care of running applications and is not the most intuitive thing to work with (although idk, maybe you're already familiar with OSes, in which case you'd know better than I would)